Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -133,6 +133,10 @@
                 <MenuFlyoutItem.Icon>
                     <FontIcon FontFamily="{StaticResource FluentUIGlyphs}" Glyph="&#xe916;" />
                 </MenuFlyoutItem.Icon>
+                <MenuFlyoutItem.KeyboardAccelerators>
```

Comment:
This event fired twice when pressing win+r. Can you fix that and check for this issue another events?

Reply:
![image](https://user-images.githubusercontent.com/20501502/97374032-0153f900-18c0-11eb-90cc-916de731f9e4.png)

Reply:
@Thomas1664

Reply:
Sorry for my late reply but I had many things to do in the last weeks.
The layout should be updated only once now.